tests/unit/conftest.py

```diff
@@ -397,18 +397,41 @@ def default_project(extra_project_params):
         grade="devel",
         parts=parts,
         license="MIT",
+        platforms={"amd64": {"build-on": ["amd64"], "build-for": ["amd64"]}},
         **extra_project_params,
     )
 
 
 @pytest.fixture()
-def default_factory(default_project):
+def default_project_core22(extra_project_params):
+    from craft_application.models import SummaryStr, VersionStr
+
+    from snapcraft.models.project import Project
+
+    parts = extra_project_params.pop("parts", {})
+
+    return Project(
+        name="default",
+        version=VersionStr("1.0"),
+        summary=SummaryStr("default project"),
+        description="default project",
+        base="core22",
```

Comment:
Is this really meant to be `core22`? As far as I know these fixtures (this and `default_factory`) are used with craft-application-based Snapcraft right?